name.abuchen.portfolio/src/name/abuchen/portfolio/online/WebLocation.java

```diff
@@ -1,13 +1,16 @@
 package name.abuchen.portfolio.online;
 
+import java.io.Serializable;
 import java.io.UnsupportedEncodingException;
 import java.net.URLEncoder;
 import java.nio.charset.StandardCharsets;
 
 import name.abuchen.portfolio.model.Security;
 
-public class WebLocation
+public class WebLocation implements Serializable
```

Comment:
Die "WebLocation" sollten wir jetzt in das "model" Package verschieben. Dadurch das "ClientSettings" im "model" Package liegt (und da gehört es auch hin) und WebLocation aus dem "online" Package braucht das wiederum das "model" Package nutzt, haben wir einen package cycle.

Ich weiß das ich im "ui" Projekt davon ein paar habe. Aber wenn die mit überschaubaren Aufwand lösbar sind, dann mache ich das auch.